lib/commands/general.js

```diff
@@ -64,6 +64,10 @@ commands.getLogTypes = function () {
   return _.keys(logTypesSupported);
 };
 
+commands.getLocationInView = async function (element) {
```

Comment:
Why is this in `general.js` while the other is in `element.js`? I would prefer both of them in `element.js`, which is the logical place for them.

Also, for consistency's sake it should be `elementId`.